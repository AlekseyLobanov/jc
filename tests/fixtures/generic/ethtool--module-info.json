{"identifier":"0x03 (SFP)","extended_identifier":"0x04 (GBIC/SFP defined by 2-wire interface ID)","connector":"0x21 (Copper pigtail)","transceiver_codes":"0x01 0x00 0x00 0x04 0x00 0x04 0x80 0xd5 0x00","transceiver_type":["Infiniband: 1X Copper Passive","Ethernet: 1000BASE-CX","Passive Cable","FC: Twin Axial Pair (TW)","FC: 1200 MBytes/sec","FC: 800 MBytes/sec","FC: 400 MBytes/sec","FC: 200 MBytes/sec","FC: 100 MBytes/sec"],"encoding":"0x00 (unspecified)","br_nominal":"10300MBd","rate_identifier":"0x00 (unspecified)","length_smf_km":"0km","length_smf":"0m","length_50um":"0m","length_62_5um":"0m","length_copper":"2m","length_om3":"0m","passive_cu_cmplnce":"0x01 (SFF-8431 appendix E) [SFF-8472 rev10.4 only]","vendor_name":"UbiquitiNetworks","vendor_oui":"00:40:20","vendor_pn":"UDC-2","vendor_rev":"","option_values":"0x00 0x00","br_margin_max":"0%","br_margin_min":"0%"}
