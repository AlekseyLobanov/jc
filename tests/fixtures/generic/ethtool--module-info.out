        Identifier                                : 0x03 (SFP)
        Extended identifier                       : 0x04 (GBIC/SFP defined by 2-wire interface ID)
        Connector                                 : 0x21 (Copper pigtail)
        Transceiver codes                         : 0x01 0x00 0x00 0x04 0x00 0x04 0x80 0xd5 0x00
        Transceiver type                          : Infiniband: 1X Copper Passive
        Transceiver type                          : Ethernet: 1000BASE-CX
        Transceiver type                          : Passive Cable
        Transceiver type                          : FC: Twin Axial Pair (TW)
        Transceiver type                          : FC: 1200 MBytes/sec
        Transceiver type                          : FC: 800 MBytes/sec
        Transceiver type                          : FC: 400 MBytes/sec
        Transceiver type                          : FC: 200 MBytes/sec
        Transceiver type                          : FC: 100 MBytes/sec
        Encoding                                  : 0x00 (unspecified)
        BR, Nominal                               : 10300MBd
        Rate identifier                           : 0x00 (unspecified)
        Length (SMF,km)                           : 0km
        Length (SMF)                              : 0m
        Length (50um)                             : 0m
        Length (62.5um)                           : 0m
        Length (Copper)                           : 2m
        Length (OM3)                              : 0m
        Passive Cu cmplnce.                       : 0x01 (SFF-8431 appendix E) [SFF-8472 rev10.4 only]
        Vendor name                               : UbiquitiNetworks
        Vendor OUI                                : 00:40:20
        Vendor PN                                 : UDC-2
        Vendor rev                                : 
        Option values                             : 0x00 0x00
        BR margin, max                            : 0%
        BR margin, min                            : 0%
